gsmphys/som_mlm.F90

```diff
@@ -157,6 +157,13 @@ subroutine ocean_init                                               &
        call abort
       endif
 
+      if (restore_method == 3 .and. .not. Model%use_ext_sst) then
+         write(6,*) ' som_mlm::ocean_init(): Cannot use restore_method == 3'
+         write(6,*) '                        unless external IC provided '
```

Comment:
This should be 'external SST' not 'external IC'

Reply:
I can make that change.